RELEASE.md

```diff
@@ -0,0 +1,56 @@
+# One App release process
+
+This defines the One App release process. 
+
+Releases can be triggered in two ways:
+
+  - [Automated release process](#automated-release-process)
+  - [Manual release process](#manual-release-process)
+
+## Automated release process
+
+ 1. An automated pull request will be raised every Wednesday at 16:00 UTC, this would be from the `prerelease` branch to master. This uses this [pull_request_release workflow](.github/workflows/pull_request_release.yml)and updates [package.json](package.json), [package-lock.json](package-lock.json), [one-app-statics package.json](one-app-statics/package.json) and the changelog(Behind the scene it uses [standard-version](https://github.com/conventional-changelog/standard-version) to generate and update this files) with the changes to be released. The commit message contains the version to be released. Updated this pull request to remove or add any new changes.
+ 2. Once a pull request is reviewed merge the pull request and please ensure that the commit message is updated to follow this pattern  
+
+   ``` bash
+   #chore(release): v5.0.0
+   chore(release): v<semantic-version>
+
+   ```
+
+   > Integration tests will continue to run in Travis as they currently do, after the automated pull request is created it would run the tests on `prerelease` branch.
+
+1. The merge will trigger the automatic generation of a new tag using the semantic version provided during the merging of the pull request above.
+
+2. After the the generated tag is pushed to the branch this will trigger the docker build and push of the images and statics to the docker hub. The development and production images would be accessible in docker [https://hub.docker.com/u/oneamex](https://hub.docker.com/u/oneamex)
```

Comment:
```suggestion
2. After the the generated tag is pushed to the branch this will trigger the docker build and publish the statics and push the images to Docker Hub. The development and production images would be accessible in docker [https://hub.docker.com/u/oneamex](https://hub.docker.com/u/oneamex)
```